gui/webdisplay/src/RWebWindow.cxx

```diff
@@ -619,7 +619,7 @@ std::string RWebWindow::GenerateKey() const
    std::string key;
 
    do {
-      key = RWebWindowsManager::GenerateKey(8);
+      key = RWebWindowsManager::GenerateKey(32);
```

Comment:
Since we now produce a long cryptographic key, we should remove the retry logic entirely.

Reply:
Just to ensure that same key is not generated twice. 
I guess, it is very low probability - but I cannot exclude this.

Reply:
The probably needs to be so slow that we should exclude it. The probably should be way below the probability of your CPU making mistakes. Otherwise it's not randomness of cryptographic quality.